src/tastytrade_sdk/market_data/market_data.py

```diff
@@ -22,7 +25,8 @@ def subscribe(self,
                   symbols: List[str],
                   on_candle: Callable[[dict], None] = None,
                   on_greeks: Callable[[dict], None] = None,
-                  on_quote: Callable[[dict], None] = None
+                  on_quote: Callable[[dict], None] = None,
+                  **kwargs,
```

Comment:
nitpick - at present, this is how `subscribe()` is auto-documented:
![image](https://github.com/user-attachments/assets/9e304ed5-665b-456f-85e8-efe36db284b5)

can we either:
- update the docstring to call out the new `aggregation_period` and `event_fields` options? or
- pass in the new options as explicit args:
```suggestion
                  aggregation_period: Optional[float] = None,
                  event_fields: Optional[dict[str, list[str]]] = None,
```
(an accompanied docstring update should also be included)